src/domain/app/i18n/fi.json

```
@@ -1883,6 +1882,39 @@
       "placeholderConfirmationMessage": "Syötä vahvistusviesti ilmoittautuneille",
       "placeholderEvent": "Valitse tai hae tapahtuma",
       "placeholderInstructions": "Syötä ilmoittautumisohjeet",
+      "registrationAccount": {
+        "buttonReturnDefaults": "Palauta tilin oletusarvot",
+        "helperDefaultValue": "Oletusarvo: {{value}}",
+        "labelAccount": "Kirjanpitotili",
+        "labelBalanceProfitCenter": "Tasapainotettu tulosyksikkö",
```

```suggestion
        "labelBalanceProfitCenter": "Tasetilin tulosyksikkö",
```
This should probably say "Tasetilin tulosyksikkö" which is the term used in the PowerPoint presentation sent by Jarkko.

---

Thanks. I was quite sure that it's not "Tasapainotettu tulosyksikkö" which was offered by Google and Deepl. Changed now